### PR TITLE
Sig 1928 kleine versie header tonen op mobiel

### PR DESCRIPTION
**Pull request by jasperswart**
In this PR:
- fixed logo size and placement in header for mobile
- removed empty grey bar on mobile 
- accepted locally by Herjen 
- the border is now now the original box-shadow from asc
- and per herjens request the site header has the same z-indexes as the admin header therefore the notification is now  "below" the site header
- all notification close buttons work in admin and in melding formulier in all devices

LET OP: kan nog niet naar ACC want de ASC moet geüpdatet worden

### CODE REVIEW

File: src/components/SiteHeader/index.js [hunk 1]
```diff
@@ -104,23 +104,31 @@ const HeaderWrapper = styled.div`
           height: 160px;
         }
         @media screen and (max-width: 539px) {
+          #header {
+            display: none;
+          }
+
           header {
-            height: 116px;
+            height: 50px;
```
Comment (janjaap) [inline src/components/SiteHeader/index.js]: Setting this to 50px will render the error notification in the wrong position on smaller screens for unauthenticated visitors:
Screenshot 2020-01-20 at 10 53 16

File: src/components/SiteHeader/index.js [hunk 2]
```diff
@@ -104,23 +104,31 @@ const HeaderWrapper = styled.div`
...
+          &:after {
+            max-width: 1400px;
+            margin-left: auto;
+            margin-right: auto;
+            content: '';
+            display: block;
+            position: absolute;
+            left: 0;
+            right: 0;
+            height: 44px;
```
[the remark this answers is not in the document]
Comment (janjaap) [reply src/components/SiteHeader/index.js]: I don't agree; `themeSpacing` should be used for margins and paddings, not heights.

File: src/components/SiteHeader/index.js (hunk 1 above)
Comment (jasperswart) [reply src/components/SiteHeader/index.js]: daar ga ik even naar kijken

File: src/components/SiteHeader/index.js [hunk 3]
```diff
@@ -104,23 +104,31 @@ const HeaderWrapper = styled.div`
...
+            margin-top: -44px;
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/components/SiteHeader/index.js]: gedaan

File: src/components/SiteHeader/index.js (hunk 1 above)
Comment (jasperswart) [reply src/components/SiteHeader/index.js]: gedaan

**Comment by janjaap** [comment]
Stacking of elements isn't going as it should:

Screenshot 2020-01-24 at 11 24 53

Screenshot 2020-01-24 at 11 27 13

File: src/components/SiteHeader/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/components/SiteHeader/index.js]: gedaan

**Comment by janjaap** [comment]
Changes don't seem to have been pushed. Please check.

**Pull request merged**